helx/charts/nginx/templates/ingress.yaml

```diff
@@ -25,4 +25,32 @@ spec:
             name: {{ include "nginx.fullname" . }}
             port:
               number: {{ .Values.service.httpPort }}
+---
+apiVersion: networking.k8s.io/v1
```

Comment:
I think this extra ingress might need to be wrapped in an `{{- if .Values.ingress.admin_ip_whitelist }}`. Because in a GKE cluster without ingress-nginx installed, this will provision an extra GCP load balancer for an extra $15 a month or so